manifests/scenarios/gmp/1_collector.yaml

```diff
@@ -172,9 +240,9 @@ spec:
         - name: config-out
           readOnly: true
           mountPath: /prometheus/config_out
-        - name: collection-secret
```

Comment:
[info] What was this for previously? Some operator thingy?

Reply:
Correct! It's actually empty if workload identity is used, so we can kill it.